crates/proto/src/serialize/txt/rdata_parsers/cert.rs

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (C) 2024 Brian Taber <btaber@zsd.systems>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+//! cert records for storing certificate data
+
+use crate::rr::rdata::CERT;
+use crate::serialize::txt::errors::ParseResult;
+
+/// Parse the RData from a set of Tokens
+#[allow(clippy::unnecessary_wraps)]
+pub(crate) fn parse<'i, I: Iterator<Item = &'i str>>(_tokens: I) -> ParseResult<CERT> {
+    // let txt_data: Vec<String> = tokens.map(ToString::to_string).collect();
```

Comment:
This doesn't look like it's complete?

Reply:
completed (looks like this was causing the DNSSEC validation issues, now my debug tests show validated on the CERT records)